### PR TITLE
ExportIndicators - Added sorting option

### PR DESCRIPTION
**Pull request by teizenman**
## Status
- [ ] In Progress
- [X] Ready
- [ ] In Hold - (Reason for hold)

Base contribution branch:
https://github.com/demisto/content/pull/11484

## Related Issues
fixes: None

## Description
Added the ability to sort exported indicators by field.

## Minimum version of Demisto
- [ ] 5.0.0
- [X] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [X] No

## Must have
- [X] Tests
- [X] Documentation

### CODE REVIEW

**Review by yaakovi — APPROVED** [review]
looks good

**Comment by teizenman** [comment]
@roysagi 
Please review the docs changes

File: Packs/ExportIndicators/Integrations/ExportIndicators/ExportIndicators.yml [hunk 1]
```diff
@@ -267,12 +267,28 @@ script:
       - 'True'
       required: false
       secret: false
+    - default: false
+      description: The field for which to sort the indicators by.
```
Comment (roysagi) [inline Packs/ExportIndicators/Integrations/ExportIndicators/ExportIndicators.yml:271]:
```suggestion
      description: The field by which to sort the indicators by.
```

File: Packs/ExportIndicators/Integrations/ExportIndicators/ExportIndicators.yml [hunk 2]
```diff
@@ -267,12 +267,28 @@ script:
...
+      isArray: false
+      name: sort_field
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: The order in which to order the indicators. The options are `asc` or `desc`.
```
Comment (roysagi) [inline Packs/ExportIndicators/Integrations/ExportIndicators/ExportIndicators.yml:278]:
```suggestion
      description: The direction by which to order the indicators. The options are `asc` or `desc`.
```

**Review by roysagi — COMMENTED** [review]
Please see my comments and change in the readme as well.

**Review by DeanArbel — APPROVED** [review]

**Pull request merged**